src/tokenization.test.ts

```diff
@@ -0,0 +1,227 @@
+import { fireEvent, waitFor } from '@testing-library/dom'
+import {
+  configureFormSubmissionMock,
+  formElementsMock,
+  formValuesMock,
+  handleFormMock,
+  malgaConfigurations,
+} from '../tests/mocks/common-configurations'
+import { MalgaTokenization } from './tokenization'
+
+vi.mock('./common/malga', async (importOriginal) => {
+  const Malga = await importOriginal<typeof import('./common/malga')>()
+  return {
+    ...Malga,
+    tokenization: vi.fn(),
+  }
+})
+const onSubmit = vi.fn()
+
+function FormForInit(onSubmit: any) {
+  const { form, holderNameInput, cvvInput, expirationDateInput, numberInput } =
+    handleFormMock()
+
+  form.setAttribute(formElementsMock.form, '')
+  form.onsubmit = onSubmit
+  form.id = 'form'
+  form.method = 'POST'
+  form.action = '/test'
+
+  holderNameInput.setAttribute(formElementsMock.holderName, '')
+  numberInput.setAttribute(formElementsMock.number, '')
+  cvvInput.setAttribute(formElementsMock.cvv, '')
+  expirationDateInput.setAttribute(formElementsMock.expirationDate, '')
+
+  document.body.appendChild(form)
+  form.appendChild(holderNameInput)
+  form.appendChild(numberInput)
+  form.appendChild(expirationDateInput)
+  form.appendChild(cvvInput)
+
+  const inputs = document.querySelectorAll('input')
+  inputs[0].value = formValuesMock.holderName
+  inputs[1].value = formValuesMock.number
+  inputs[2].value = formValuesMock.expirationDate
+  inputs[3].value = formValuesMock.cvv
+}
+function FormForTokenize() {
+  const { form, holderNameInput, cvvInput, expirationDateInput, numberInput } =
+    handleFormMock()
+
+  form.setAttribute(formElementsMock.form, '')
+
+  form.id = 'form'
+  form.method = 'POST'
+  form.action = '/test'
+
+  holderNameInput.setAttribute(formElementsMock.holderName, '')
+  numberInput.setAttribute(formElementsMock.number, '')
+  cvvInput.setAttribute(formElementsMock.cvv, '')
+  expirationDateInput.setAttribute(formElementsMock.expirationDate, '')
+
+  document.body.appendChild(form)
+  form.appendChild(holderNameInput)
+  form.appendChild(numberInput)
+  form.appendChild(expirationDateInput)
+  form.appendChild(cvvInput)
+
+  const inputs = document.querySelectorAll('input')
+  inputs[0].value = formValuesMock.holderName
+  inputs[1].value = formValuesMock.number
+  inputs[2].value = formValuesMock.expirationDate
+  inputs[3].value = formValuesMock.cvv
+}
+describe('init', () => {
```

Comment:
Aqui ta fora do padrão que comentamos no review passado:

```suggestion
describe('MalgaTokenization', () => {
  describe('init', () => {
     test('should...', () => {})
  }),
  describe('tokenize', () => {
     test('should...', () => {})
  })
})
```